src/Lightning/NoiseProtocol/LightningNetworkConfig.cs

```diff
@@ -0,0 +1,30 @@
+using System.Text;
+
+namespace NoiseProtocol
+{
+   public static class LightningNetworkConfig
+   {
+      const string PROTOCOL_NAME = "Noise_XK_secp256k1_ChaChaPoly_SHA256";
+      private const string PROLOGUE = "lightning";
+
+      public static byte[] ProtocolNameByteArray()
+      {
+         var byteArray = new byte[PROTOCOL_NAME.Length];
```

Comment:
stackalloc maybe?

Reply:
Will move all the lightning config logic to read from json file as we talked, in a later task.